spec/controllers/hyrax/batch_edits_controller_spec.rb

```diff
@@ -35,20 +37,27 @@
     let!(:two) do
       create(:generic_work, creator: ["Fred"], title: ["abc"], language: ['en'], user: user)
     end
+
+    let!(:three) do
+      create(:generic_work, creator: ["Fred"], title: ["abc"], language: ['en'])
+    end
+
     let(:mycontroller) { "hyrax/my/works" }
 
     before do
       # TODO: why aren't we just submitting batch_document_ids[] as a parameter?
-      controller.batch = [one.id, two.id]
+      controller.batch = [one.id, two.id, three.id]
       expect(controller).to receive(:can?).with(:edit, one.id).and_return(true)
       expect(controller).to receive(:can?).with(:edit, two.id).and_return(true)
+      expect(controller).to receive(:can?).with(:edit, three.id).and_return(false)
     end
 
     it "is successful" do
       put :update, params: { update_type: "delete_all" }
       expect(response).to redirect_to(dashboard_path(locale: 'en'))
       expect { GenericWork.find(one.id) }.to raise_error(Ldp::Gone)
       expect { GenericWork.find(two.id) }.to raise_error(Ldp::Gone)
+      expect(GenericWork.find(three.id).id).to eq(three.id)
```

Comment:
This should be `expect(GenericWork).to exist(three.id)` (or alternately `expect(GenericWork.exists?(three.id).to be true`)